ARM-wvd-templates/CreateAndProvisionHostPool/README.md

```diff
@@ -1,15 +1,15 @@
 Click the button below to deploy:
 
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FCreate%20and%20provision%20WVD%20host%20pool%2FCreateHostpoolTemplate.json" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FCreateAndProvisionHostPool%2FCreateHostpoolTemplate.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
-<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FCreate%20and%20provision%20WVD%20host%20pool%2FCreateHostpoolTemplate.json" target="_blank">
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FCreateAndProvisionHostPool%2FCreateHostpoolTemplate.json" target="_blank">
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
 # ARM Template to Create and provision new Windows Virtual Desktop hostpool
 
-This ARM template only supports Azure Resource Manager Windows Virtual Desktop objects, and does not support Windows Virtual Desktop (classic). Link to the v1 readme [here](https://github.com/Azure/RDS-Templates/tree/master/wvd-templates/Create%20and%20provision%20WVD%20host%20pool)

```

Comment:
This should not be changed lol Its pointing to v1
Also you may want to review my other comment here: https://github.com/Azure/RDS-Templates/pull/496#discussion_r464712972

Reply:
I fixed the links to the v1 in the readme, the change is in staging and will go live at 12pm today.

Reply:
thanks marcel

Reply:
the template can have multiple rgs so Im going to keep it explicit.